test/unit/support.js

```diff
@@ -231,7 +259,8 @@ testIframeWithCallback( "Check CSP (https://developer.mozilla.org/en-US/docs/Sec
 			"pixelMarginRight": false,
 			"pixelPosition": false,
 			"radioValue": true,
-			"reliableMarginRight": true
+			"reliableMarginRight": true,
+			"reliableDimensionsInFullscreenMode": true
```

Comment:
Can you put it before `reliableMarginRight` everywhere? We're trying to keep the list alphabetically sorted. :)